telco-examples/edge-clusters/cpu-manager/README.md

```diff
@@ -0,0 +1,45 @@
+# Example of Edge Cluster with CPU Manager
+
+## Introduction
+
+This is an example to demonstrate how to deploy an edge cluster with cpu manager enabled in kubernetes for Telco using SUSE ATIP and the fully automated directed network provisioning.
+
+## Configuration
+
+Using the file `telco-examples/edge-clusters/cpu-manager/edge-cluster-cpu-manager.yaml` as an example, you can deploy an edge cluster with cpu manager just modifying the following parameters:
```

Comment:
Probably "**by** just modifying" or "after defining", something like that.

Reply:
I prefer "by modifying" because "just" can read as condescending

Reply:
Changed! Thanks